src/components/organisms/CompareModal/CompareActionBar.tsx

```diff
@@ -70,6 +63,61 @@ export const CompareActionBar: React.FC = () => {
   );
 };
 
+function SearchInput({disabled}: {disabled: boolean}) {
```

Comment:
Probably I'll get a comment to split this up, but the total file length is now 170 lines which is a decent length  imo. The compareModal directory is getting rather big and I feel like splitting this up reduces readability instead of improving it. Rather personal (and maybe controversial) opinion so let me know what you think!

Reply:
I saw it but looks fine, it's nice that you have it under the main component.